WebUI/ConestogaInsidersClub.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Dapper" Version="2.0.90" />
```

Comment:
Dapper is an alternative ORM to EF Core. I don't think we need it in this project.